<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="griffon.core.i18n.NoSuchMessageExceptionSpec" tests="2" skipped="0" failures="0" errors="0" timestamp="2016-06-01T20:37:40" hostname="testing-gce-02346492-7fe8-4844-953c-74fc61fd1164.c.travis-ci-prod-5.internal" time="0.005">
  <properties/>
  <testcase name="Exercise constructor with errors" classname="griffon.core.i18n.NoSuchMessageExceptionSpec" time="0.004"/>
  <testcase name="Successfully create exception" classname="griffon.core.i18n.NoSuchMessageExceptionSpec" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
